daml-lf/transaction/src/test/scala/com/digitalasset/daml/lf/validation/ValidationSpec.scala

```diff
@@ -542,45 +543,31 @@ class ValidationSpec extends AnyFreeSpec {
   //--[per tweak tests]--
 
   "Significant tweaks" - {
-    var count = 0
     significantTweaks.foreach { case (name, tweak) =>
```

Comment:
This also ought to be `forEvery`'d; nesting works fine. But no hurry anyway

Reply:
I checked the _Table-driven property checks_ docs: https://www.scalatest.org/user_guide/table_driven_property_checks
Do you mean `forAll` ?
What's the advantage over a vanilla `foreach`?

Reply:
No, `forEvery`. As discussed over here https://github.com/digital-asset/daml/pull/10167#discussion_r664356959